src/fileAnalysis/AnalyzeFile.java

```diff
@@ -8,9 +8,14 @@ public static void main(String [] args) {
     public static void analyzeFile() {
         try {
             String text = ReadFile.readFile("src/resources/SampleText.txt");
+<<<<<<< HEAD
```

Comment:
Hmmm it looks like the merge conflict snuck in. Could you fix it again (just go into VS code and remove the <<<< HEAD, ====, and the >>>>>a9..., add the file AnalyzeFile.java, commit the change and push the branch again